batch/batches/ExportCsv/Engine/KObjectExportEngine.php

```diff
@@ -8,7 +8,7 @@ abstract class KObjectExportEngine
 {
 	/**
 	 * @param int $objectType of enum KalturaCopyObjectType
-	 * @return KCopyingEngine
+	 * @return KUserExportEngine
```

Comment:
@TalDubovKaltura this was originally my bad - could you please change the return type here to "KObjectExportEngine"? We're not necessarily retuning a user exporting engine.

Reply:
@hilak Done. actually it was my bad too because I didn't completely interpret which type of value the method returns, and it should be the parent class like you wrote.